UnitTests/Filters/StringFiltersTests.swift

```diff
@@ -12,13 +12,26 @@ class StringFiltersTests: XCTestCase {
   func testLowerFirstWord() {
     let expectations = [
       "string": "string",
-      "string_with_words": "string_with_words",
+      "String": "string",
+      "strIng": "strIng",
+      "strING": "strING",
+      "X": "x",
+      "x": "x",
       "SomeCapString": "someCapString",
+      "someCapString": "someCapString",
+      "string_with_words": "string_with_words",
+      "String_with_words": "string_with_words",
+      "String_With_Words": "string_With_Words",
+      "STRing_with_words": "stRing_with_words",
+      "string_wiTH_WOrds": "string_wiTH_WOrds",
       "": "",
-      "X": "x",
       "URLChooser": "urlChooser",
+      "a__b__c": "a__b__c",
       "__y_z!": "__y_z!",
-      "PLEASESTOPSCREAMING": "pleasestopscreaming"
+      "PLEASESTOPSCREAMING": "pleasestopscreaming",
+      "PLEASESTOPSCREAMING!": "pleasestopscreaminG!",
```

Comment:
Why would this case upper-case the last `G`?
Same for the next two expectations.

Maybe that's how the filter works now, so the unit tests were written to match them, but that's not what we want as an output in those cases, so unit tests should dictate what we want and implementation should be written so as to match the expectations